pkg/controllers/nodepool/counter/suite_test.go

```diff
@@ -180,26 +200,46 @@ var _ = Describe("Counter", func() {
 		nodePool = ExpectExists(ctx, env.Client, nodePool)
 
 		// Should equal both the nodeClaim and node capacity
-		Expect(nodePool.Status.Resources).To(BeEquivalentTo(nodeClaim2.Status.Capacity))
-		Expect(nodePool.Status.Resources).To(BeEquivalentTo(node2.Status.Capacity))
+		expected = resetResource()
+		expected = resources.MergeInto(expected, nodeClaim2.Status.Capacity)
+		expected[v1.ResourceName("nodes")] = resource.MustParse("1")
+		Expect(nodePool.Status.Resources).To(BeComparableTo(expected))
+		expected = resetResource()
+		expected = resources.MergeInto(expected, node2.Status.Capacity)
+		expected[v1.ResourceName("nodes")] = resource.MustParse("1")
+		Expect(nodePool.Status.Resources).To(BeComparableTo(expected))
 	})
-	It("should nil out the counter when all nodes are deleted", func() {
+	It("should zero out the counter when all nodes are deleted", func() {
 		ExpectApplied(ctx, env.Client, node, nodeClaim)
 		ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx, env.Client, nodeController, nodeClaimController, []*v1.Node{node}, []*v1beta1.NodeClaim{nodeClaim})
 
 		ExpectObjectReconciled(ctx, env.Client, nodePoolController, nodePool)
 		nodePool = ExpectExists(ctx, env.Client, nodePool)
 
 		// Should equal both the nodeClaim and node capacity
-		Expect(nodePool.Status.Resources).To(BeEquivalentTo(nodeClaim.Status.Capacity))
-		Expect(nodePool.Status.Resources).To(BeEquivalentTo(node.Status.Capacity))
+		expected := resources.MergeInto(expected, nodeClaim.Status.Capacity)
+		expected[v1.ResourceName("nodes")] = resource.MustParse("1")
+		Expect(nodePool.Status.Resources).To(BeComparableTo(expected))
+		expected = resetResource()
+		expected = resources.MergeInto(expected, node.Status.Capacity)
+		expected[v1.ResourceName("nodes")] = resource.MustParse("1")
+		Expect(nodePool.Status.Resources).To(BeComparableTo(expected))
 
 		ExpectDeleted(ctx, env.Client, node, nodeClaim)
 
 		ExpectReconcileSucceeded(ctx, nodeController, client.ObjectKeyFromObject(node))
 		ExpectReconcileSucceeded(ctx, nodeClaimController, client.ObjectKeyFromObject(nodeClaim))
 		ExpectObjectReconciled(ctx, env.Client, nodePoolController, nodePool)
 		nodePool = ExpectExists(ctx, env.Client, nodePool)
-		Expect(nodePool.Status.Resources).To(BeNil())
+		expected = resetResource()
+		Expect(nodePool.Status.Resources).To(BeComparableTo(expected))
 	})
 })
+
+func resetResource() v1.ResourceList {
```

Comment:
```suggestion
func baseResources() v1.ResourceList {
```

Maybe `baseResources` or `zeroResources`? Does this get easier if this is a statically configured `v1.ResourceList` rather than having to regeenerate this every time?